### PR TITLE
HKG: Genesis GV70 2023 support

### PR DESCRIPTION
**Pull request by heemanshoe**
```CAR.GENESIS_GV70_1ST_GEN: {
    (Ecu.fwdCamera, 0x7c4, None): [
      b'\xf1\x00JK1 MFC  AT USA LHD 1.00 1.04 99211-AR000 210204',
      b'\xf1\x00JK1 MFC  AT USA LHD 1.00 1.01 99211-AR200 220125',
      b'\xf1\x00JK1 MFC  AT USA LHD 1.00 1.01 99211-AR300 220125',
    ],
    (Ecu.fwdRadar, 0x7d0, None): [
      b'\xf1\x00JK1_ SCC FHCUP      1.00 1.02 99110-AR000         ',
      b'\xf1\x00JK1_ SCC FHCUP      1.00 1.00 99110-AR200         ',
      b'\xf1\x00JK1_ SCC FHCUP      1.00 1.00 99110-AR300         ',
    ],
  }
```

Only other part that I changed in values was the harness line. I didn't run through the code to see if that mattered anywhere but my connector was different than what others had used for my model so I changed it in values.py just to rule out any possible issues.

Dongle ID: 927df12cb3573fcd
VIN: KMUMCDTC4PU120843

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -248,7 +248,7 @@ def init_make(self, CP: car.CarParams):
   ],
   CAR.GENESIS_G70: HyundaiCarInfo("Genesis G70 2018-19", "All", harness=Harness.hyundai_f),
   CAR.GENESIS_G70_2020: HyundaiCarInfo("Genesis G70 2020", "All", harness=Harness.hyundai_f),
-  CAR.GENESIS_GV70_1ST_GEN: HyundaiCarInfo("Genesis GV70 2022-23", "All", harness=Harness.hyundai_l),
+  CAR.GENESIS_GV70_1ST_GEN: HyundaiCarInfo("Genesis GV70 2022-23", "All", harness=Harness.hyundai_m),
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:251]: Quickly searched Discord and didn't see any other users complain of this harness not working, so let's try to figure that out first before we change.

**Comment by sshane** [comment]
Looks like you're the only 3.5T for any model year! The GV60 also has a very similar harness split (Performance and Advanced trims) with the harness connector being flipped horizontally, but otherwise the same for the higher and lower trims for both cars.

**Comment by heemanshoe** [comment]
> Looks like you're the only 3.5T for any model year! The GV60 also has a very similar harness split (Performance and Advanced trims) with the harness connector being flipped horizontally, but otherwise the same for the higher and lower trims for both cars.

FYI, I believe there was a GV80 3.5T (BobaJohn#7056).

**Review by heemanshoe — COMMENTED** [review]
I think you're missing a comma here on line 254

**Comment by sshane** [comment]
Yep, meant first 3.5T for the GV70. I'll reach out to him to see which harness he got.

**Pull request merged**